cocotb/xunit_reporter.py

```diff
@@ -85,30 +53,6 @@ def add_property(self, testsuite=None, **kwargs):
         self.last_property = SubElement(testsuite, "property", **kwargs)
         return self.last_property
 
-    def update_testsuite(self, testsuite=None, **kwargs):
-        if testsuite is None:
-            testsuite = self.last_testsuite
-        for k in kwargs:
-            testsuite.set(k, str(kwargs[k]))
-
-    def update_testsuites(self, **kwargs):
-        for k in kwargs:
-            self.results.set(k, str(kwargs[k]))
-
-    def add_log(self, logfile, testcase=None):
-        if testcase is None:
-            testcase = self.last_testcase
-        log = SubElement(testcase, "system-out")
-        f = File(logfile, 'r+')
-        lines = f.countlines()
-        if lines > (TRUNCATE_LINES * 2):
-            head = f.head(TRUNCATE_LINES)
-            tail = f.tail(TRUNCATE_LINES)
-            log.text = "".join(head + list("[...truncated %d lines...]\n" %
-                               ((lines - (TRUNCATE_LINES*2)))) + tail)
-        else:
-            log.text = "".join(f.readlines())
```

Comment:
It would be cool if the log lines actually did end up in the xunit output - but perhaps the approach used here isn't useful.